<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionclient</class>
 <widget class="QDialog" name="gestionclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_client">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1011</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Interface">
    <attribute name="title">
     <string>Interface</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IDENTIFIANT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PRENOM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ADRESSE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NUMERO DE TELEPHONE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_afficher">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_adresse">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_num">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifer">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>460</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_interface">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>611</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>             RECHERCHE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_recherche">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>pushButton_afficher</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton_ajouter</zorder>
    <zorder>pushButton_supprimer</zorder>
    <zorder>lineEdit_id</zorder>
    <zorder>lineEdit_nom</zorder>
    <zorder>lineEdit_prenom</zorder>
    <zorder>lineEdit_adresse</zorder>
    <zorder>lineEdit_num</zorder>
    <zorder>pushButton_modifer</zorder>
    <zorder>tableView_interface</zorder>
    <zorder>label_6</zorder>
    <zorder>lineEdit_recherche</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QTableView" name="tableView_client">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>651</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tri_id">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>460</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tri_nom">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>460</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri Nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tri_prenom">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>460</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri Prenom</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Abonnement</string>
    </attribute>
    <widget class="QStackedWidget" name="stackedWidget_ab">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>491</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>IDENTIFIANT CLIENT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NUMERO D'ABONNEMENT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE DE DEBUT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE D'EXPIRATION</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>NOMBRE DE POINT FIDELITE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifer_ab">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>460</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimer_ab">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>460</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_afficher_ab">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>460</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_ab">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date_deb">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date_fin">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>280</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_point">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>360</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter_ab">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>460</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_ab">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>571</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_suivant">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>460</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Suivant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_num_ab">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>             RECHERCHE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rech_ab">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QTableView" name="tableView_ab_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>951</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_id_ab">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>440</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Trie ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_ab">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>440</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Trie AB</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_point_ab">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>440</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Trie point</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_retour_ab">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>440</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
